common/list/lists-mws-en.html

```diff
@@ -0,0 +1,599 @@
+		&lt;li&gt;List item 3&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;
+&lt;div <strong>class=&quot;lst-spcd&quot;</strong>&gt;
+	&lt;ol&gt;
+		&lt;li&gt;List item 1&lt;/li&gt;
+		&lt;li&gt;List item 2&lt;/li&gt;
+		&lt;li&gt;List item 3&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/div&gt;
+</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Default markup - <span class="label label-success mrgn-lft-sm">Recommended</span></h3>
+		<ul class="lst-spcd">
+			<li>List item 1</li>
+			<li>List item 2</li>
+			<li>List item 3</li>
+		</ul>
+		<ol class="lst-spcd">
+			<li>List item 1</li>
+			<li>List item 2</li>
+			<li>List item 3</li>
+		</ol>
+		<h4>Code</h4>
+<pre><code>// Spaced list:
+&lt;ul <strong>class=&quot;lst-spcd&quot;</strong>&gt;
+	&lt;li&gt;List item 1&lt;/li&gt;
+	&lt;li&gt;List item 2&lt;/li&gt;
+	&lt;li&gt;List item 3&lt;/li&gt;
+&lt;/ul&gt;
+&lt;ol <strong>class=&quot;lst-spcd&quot;</strong>&gt;
+	&lt;li&gt;List item 1&lt;/li&gt;
+	&lt;li&gt;List item 2&lt;/li&gt;
+	&lt;li&gt;List item 3&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>List double spaced <code>.lst-spcd-2</code></h2>
+	<div class="col-md-6">
+		<h3>Custom markup</h3>
+		<div class="lst-spcd-2">
+			<ul>
+				<li>Item</li>
+				<li>Item</li>
+				<li>Item</li>
+				<li>Item</li>
+				<li>Item</li>
+			</ul>
+		</div>
+		<div class="lst-spcd-2">
+			<ol>
+				<li>Item</li>
+				<li>Item</li>
+				<li>Item</li>
+				<li>Item</li>
+				<li>Item</li>
+			</ol>
+		</div>
+		<h4>Code</h4>
+<pre><code>&lt;div <strong>class=&quot;lst-spcd-2&quot;</strong>&gt;
+	&lt;ul&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;
+&lt;div <strong>class=&quot;lst-spcd-2&quot;</strong>&gt;
+	&lt;ol&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+		&lt;li&gt;Item&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/div&gt;</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Default markup - <span class="label label-success mrgn-lft-sm">Recommended</span></h3>
+		<ul class="lst-spcd-2">
+			<li>Item</li>
+			<li>Item</li>
+			<li>Item</li>
+			<li>Item</li>
+			<li>Item</li>
+		</ul>
+		<ol class="lst-spcd-2">
+			<li>Item</li>
+			<li>Item</li>
+			<li>Item</li>
+			<li>Item</li>
+			<li>Item</li>
+		</ol>
+		<h4>Code</h4>
+<pre><code>&lt;ul <strong>class=&quot;lst-spcd-2&quot;</strong>&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+&lt;/ul&gt;
+&lt;ol <strong>class=&quot;lst-spcd-2&quot;</strong>&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+	&lt;li&gt;Item&lt;/li&gt;
+&lt;/ol&gt;
+</code></pre>
+	</div>
+</div>
+
+<div class="row">
+	<h2>Custom CSS for list styling via parent</h2>
+	<div class="col-md-6">
+		<h3>Custom markup</h3>
```

Comment:
This heading do have sibling heading that have exactly the same content, this create a confusion and the WCAG SC 2.4.6 are not satisfied.

```suggestion
		<h3>Disc styled by its parent container</h3>
```

Reply:
Changed. Thanks